### ISSUE TITLE
It's an interesting project. Could you provide some examples to use it?

### ISSUE THREAD

**Issue by taoyuan** [uditalias/react-os#1]
I'm doing something like this in an embedded environment.It's a great idea to realize a `react-os`. 

I really want to know that `react-os` how to work. Would you please provide one or some examples to show how to use it?

**Comment by agupta1989** [comment]
@taoyuan Yeah, It looks promising, but I see no active development from last 2 years !

**Comment by taoyuan** [comment]
@agupta1989 This is really regrettable. Maybe the author has lost interest on it. 

Do you know that there are other similar projects?

**Comment by agupta1989** [comment]
@taoyuan well, if you want to develop desktop based applications using JS stack, electron is way to go, I have not idea about any other react based library.

**Comment by uditalias** [comment]
Hi guys
Glad to see you're interested in this project.
Im very sory but i dont have so mutch time to work on it.
@agupta1989 the idea is to make web applications that looks like operating systems inside the browser.

**Comment by agupta1989** [comment]
@uditalias ah.. got it. so you mean webtops ;) just like what qooxdoo or Sencha's ExtJS is capable of !
